Content.Client/Stylesheets/Redux/BaseStylesheet.Sheetlets.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Robust.Client.UserInterface;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Stylesheets.Redux;
+
+public abstract partial class BaseStylesheet
+{
+    public StyleRule[] GetSheetletRules<TSheetTy>(Type sheetletTy)
```

Comment:
maybe it could be given a better name, but `[CommonSheetlet]` makes the sheetlet accessible to the stylesheets, but all the valid ones should be loaded by the stylesheets. if it is not loaded it is because it does not meet the type constaints. 

That's a good idea though, I'll add a check that will log any `[CommonSheetlet]`s that are not loaded by any stylesheet.